### ISSUE TITLE
[Version: 1.1.6] Aspect with composition

### ISSUE THREAD

**Issue by WAVIFICATION** [cap-js/attachments#100]
**Describe the bug**
If the schema contains an aspect with composition, `@cap-js/attachments` crashes.

**To Reproduce**
Add the following to a sap cap project

```
using {Attachments} from '@cap-js/attachments';

service SampleService {
  entity element: common.Attribute {
    key ID : String;
    title : String;
  }
}

aspect common.Attribute {
    to_attributes : Composition of many AttributeTemplate;
}
aspect AttributeTemplate {
    text : String;
}
```

**Expected behavior**
Should not crash

- [ ] is it a regression issue? 

**Screenshots**
![image](https://github.com/user-attachments/assets/ea5fbc0f-9af0-4f87-956d-594d4ff7805b)

**Customer Info**
Company: Gravity-IT Pty Ltd.

**Comment by WAVIFICATION** [comment]
Thanks!!

**Comment by muskansethi1** [comment]
Hi @WAVIFICATION,
The fix for this issue is available in Version 1.1.7

Best Regards,
Muskan Sethi
